terraform/modules/paas/app-catalog-service.tf

```diff
@@ -1,6 +1,29 @@
 resource "cloudfoundry_user_provided_service" "app_catalog" {
   name  = "app-catalog"
   space = data.cloudfoundry_space.space.id
+  credentials = {
```

Comment:
This could be simplified a little by adding the credentials map to a local variable e.g.

```
locals {
  app_catalog_urls = {
    "adminusers_url" ...
  }
}
```

Then pass it into the `credentials = local.app_catalog_urls`

Reply:
yeah that's a good idea...hold on I'll change it.